.gitmodules

```diff
@@ -32,3 +32,11 @@
 	path = extern/evapotranspiration/evapotranspiration
 	url = https://github.com/NOAA-OWP/evapotranspiration.git
         branch = master
+[submodule "extern/SoilFreezeThaw/SoilFreezeThaw"]
+	path = extern/SoilFreezeThaw/SoilFreezeThaw
+	url = https://github.com/NOAA-OWP/SoilFreezeThaw
+        branch = master
+[submodule "extern/SoilMoistureProfiles/SoilMoistureProfile"]
+	path = extern/SoilMoistureProfiles/SoilMoistureProfile
+	url = https://github.com/NOAA-OWP/SoilMoistureProfiles
```

Comment:
This is likely to lead to confusion... let's make these all `SoilMoistureProfiles` (with an `s`), since that is apparently the name of the repo.

Reply:
Notably, the submodule addition itself probably needs to be changed--I am not sure that just changing the path in this file will suffice...and I'm not sure how to otherwise edit it, it may need to be removed and re-added. (?)

Reply:
I removed and re-added SoilMoistureProfiles.